src/uikit/Select/ui.js

```diff
@@ -0,0 +1,46 @@
+import { css } from 'emotion';
+import styled from 'react-emotion';
+
+export const OptionDropdownWrapper = styled('div')`
+  position: absolute;
+  background: white;
+  min-width: 100%;
+  z-index: 1;
+  border: 1px solid rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
+  cursor: pointer;
+  padding: 5px;
+  top: 100%;
+  right: ${({ align }) => (align === 'right' ? `0` : `auto`)};
+  left: ${({ align }) => (align === 'right' ? `auto` : `0`)};
+  ${({ className }) =>
```

Comment:
no need for this right? `styled` should pass `className` through as a prop & just apply it to the `div`